customskill/skill.go

```diff
@@ -0,0 +1,90 @@
+package customskill
+
+import (
+	"encoding/json"
+	"io"
+	"reflect"
+
+	"github.com/mikeflynn/go-alexa/customskill/request"
+	"github.com/mikeflynn/go-alexa/customskill/response"
+	"github.com/pkg/errors"
+)
+
+type Skill struct {
+	ValidApplicationIDs []string
+	OnLaunch            func(*request.LaunchRequest) (*response.Envelope, error)
+	OnIntent            func(*request.IntentRequest, *request.Session) (*response.Envelope, error)
+	OnSessionEnded      func(endedRequest *request.SessionEndedRequest) error
```

Comment:
did you mean to have the `endedRequest` variable name here? or just the parameter type?

Reply:
Maybe? I'll take a look